### DISCUSSION TITLE
Feature Request: Auto-Cleanup of Empty Folders After Trailer Deletion

### DISCUSSION THREAD

**Discussion by alexkahler** [nandyalu/trailarr#49] [Ideas]
I've noticed that when I delete trailers for unreleased movies (where no movie file exists on the drive and Radarr hasn't created a folder yet), the folder created by Trailarr is left behind. This results in quite a mess in my otherwise organized Movies folder, which is a bit of an OCD pet peeve for me.

Could we please have an option for Trailarr to delete the folder if it's empty when a trailer file is removed? It would be a great way to ensure folders are automatically cleaned up and maintain a tidy directory.

**Comment by nandyalu** [comment]
I think there was a bug in earlier versions of the app that prevented it from deleting the empty trailer folder. Trailarr should be deleting the empty Trailers folder when deleting the trailer.

Are you still seeing this behavior?
